dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResource.java

```diff
@@ -240,119 +243,122 @@ public final Response containerContent(@Context final HttpServletRequest req, @C
-            .build();
+        try {
+            final Language id = WebAPILocator.getLanguageWebAPI()
+                    .getLanguage(req);
+
+            ShortyId contentShorty = APILocator.getShortyAPI()
+                    .getShorty(contentletId)
+                    .orElseGet(() -> {
+                        throw new ResourceNotFoundException(
+                                "Can't find contentlet:" + contentletId);
+                    });
+            final Contentlet contentlet =
+                    (contentShorty.subType == ShortType.CONTENTLET) ? APILocator.getContentletAPI()
+                            .find(contentShorty.longId, user, !mode.isAdmin)
+                            : APILocator.getContentletAPI()
+                                    .findContentletByIdentifier(contentShorty.longId, mode.showLive,
+                                            id.getId(), user, !mode.isAdmin);
+
+            ShortyId containerShorty = APILocator.getShortyAPI()
+                    .getShorty(containerId)
+                    .orElseGet(() -> {
+                        throw new ResourceNotFoundException("Can't find Container:" + containerId);
+                    });
+            Container container =
+                    (containerShorty.subType == ShortType.CONTAINER) ? APILocator.getContainerAPI()
+                            .find(containerId, user, !mode.isAdmin)
+                            : (mode.showLive) ? (Container) APILocator.getVersionableAPI()
+                                    .findLiveVersion(containerShorty.longId, user, !mode.isAdmin)
+                                    : (Container) APILocator.getVersionableAPI()
+                                            .findWorkingVersion(containerShorty.longId, user,
+                                                    !mode.isAdmin);
+
+            APILocator.getPermissionAPI()
+                    .checkPermission(contentlet, PermissionLevel.READ, user);
+            APILocator.getPermissionAPI()
+                    .checkPermission(container, PermissionLevel.EDIT, user);
+
+            MultiTree mt = new MultiTree().setContainer(containerId)
+                    .setContentlet(contentletId)
+                    .setRelationType(uid);
+
+            MultiTreeFactory.deleteMultiTree(mt);
+
+            return Response.ok("ok")
+                    .build();
+        } catch (DotSecurityException e) {
+            throw new ForbiddenException(e);
+        }
     }
 
 
 
     @Path("/containerContent/{params:.*}")
     public final Response containerContents(@Context final HttpServletRequest req, @Context final HttpServletResponse res,
             @QueryParam("containerId") final String containerId, @QueryParam("contentInode") final String contentInode)
-            throws DotDataException, DotSecurityException, ParseErrorException, MethodInvocationException,
+            throws DotDataException, ParseErrorException, MethodInvocationException,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.apache.velocity.exception.ParseErrorException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.apache.velocity.exception.MethodInvocationException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)